scripts/upload-snapshot

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# please don't share `~/.bintray.conf` to VCS since it must contain credentials as below:
```

Comment:
@cbeust 
please take care of this, can prepare one conf before you do some test.

see how you can get the api token of bintray: https://bintray.com/docs/usermanual/interacting/interacting_editingyouruserprofile.html#anchorAPIKEY